buildcfg.txt

```diff
@@ -2456,10 +2456,10 @@ PLSGB0	-50	-100
 PLSSA0	9	12
 PLSSB0	9	12
 POL1A0	22	62
-POL2A0	19	62
```

Comment:
The Y offsets are all at the very bottom of the image even though the sprite is drawn with a vertically extending pool of gore. To look best in vanilla the offset should be moved up by 5:

![DOOM03](https://github.com/freedoom/freedoom/assets/24984517/f9a26b97-fafb-4b3f-8e6b-12a6a3316022)
![DOOM02](https://github.com/freedoom/freedoom/assets/24984517/bc51ba0e-92bc-4585-89ec-b26cbe3b7ec2)

Reply:
So the Y offset should be 67 or 57?

Reply:
The "raised offset" image has 43, 53 would have the sprite unequivocally hovering in the air.

Just one warning, though, these do get clipped into the ground in GZDoom so the bottom part is invisible. It's a tradeoff but personally I think it looks less worse clipped in GZDoom than hovering in vanilla or any sourceport:
![_](https://github.com/freedoom/freedoom/assets/24984517/d6a74d83-7889-492a-a6c0-4f806e9a6572)

Reply:
I just found out that GZDoom has a new "Smarter" option for "Adjust sprite clipping" while I was testing that out myself.